qiskit_nature/exceptions.py

```diff
@@ -19,3 +19,9 @@ class QiskitNatureError(QiskitError):
     """Class for errors returned by Qiskit's Nature module."""
 
     pass
+
+
+class NotSupportedMethodError(QiskitNatureError):
```

Comment:
Just an opinion: I think `UnsupportMethodError` could read more nicely. What do you think?

Reply:
Renamed